example.js

```diff
@@ -0,0 +1,9 @@
+const plugin = require('.')()
+
+async function run () {
+  await plugin.connect()
+  await plugin.sendData(/* ... */)
```

Comment:
Is this example intended to be runnable? If so, there should be some dummy data here.